app/models/rejection.rb

```diff
@@ -37,6 +37,9 @@ class Rejection < ActiveRecord::Base
   validates :reasons, presence: true
   validate :validate_allowance_renews_on_date
 
+  before_save do
```

Comment:
Ah, I think I haven't explained myself well! Not saying that we should save duplicates, just keep the reasons uniq as it is modified